Cql/Elm/IDefinitionElement.cs

```diff
@@ -171,23 +173,18 @@ public partial class ExpressionDef : IDefinitionElement
 
         IDefinitionElement IDefinitionElement.AddError(CqlToElmError error) => this.AddError(error);
 
+        public override string ToString() => $" {Name} : {resultTypeSpecifier}";
```

Comment:
The space before {Name} can be removed. If there was a reason for that space here, its a code smell.